.ci/on-schedule.sh

```diff
@@ -216,7 +274,8 @@ for package in "${PACKAGES[@]}"; do
 done
 
 if [ ${#MODIFIED_PACKAGES[@]} -ne 0 ]; then
-    "$(dirname "$(realpath "$0")")"/schedule-packages.sh "${MODIFIED_PACKAGES[@]}"
+    "$(dirname "$(realpath "$0")")"/schedule-packages.sh "${MODIFIED_PACKAGES[*]}"
+    "$(dirname "$(realpath "$0")")"/manage-aur.sh "${MODIFIED_PACKAGES[*]}"
```

Comment:
Why switch from "${MODIFIED_PACKAGES[@]}" to "${MODIFIED_PACKAGES[*]}"?

```suggestion
    "$(dirname "$(realpath "$0")")"/schedule-packages.sh "${MODIFIED_PACKAGES[@]}"
    "$(dirname "$(realpath "$0")")"/manage-aur.sh "${MODIFIED_PACKAGES[@]}"
```